demisto_sdk/dev_tools/lint_manager.py

```diff
@@ -0,0 +1,139 @@
+import os
+import sys
+import subprocess
+import concurrent.futures
+from typing import Tuple
+from demisto_sdk.common.configuration import Configuration
+from demisto_sdk.common.tools import print_color, LOG_COLORS
+
+from demisto_sdk.dev_tools.linter import Linter
+
+
+class LintManager:
+    def __init__(self, project_dir: str, no_test: bool = False, no_pylint: bool = False, no_flake8: bool = False,
+                 no_mypy: bool = False, verbose: bool = False, root: bool = False, keep_container: bool = False,
+                 cpu_num: int = 0, parallel: bool = False, configuration: Configuration = Configuration()):
+
+        if no_test and no_pylint and no_flake8 and no_mypy:
+            raise ValueError("Nothing to run as all --no-* options specified.")
+
+        self.parallel = parallel
+        self.log_verbose = verbose
+        self.root = root
+        self.parallel = parallel
+        self.keep_container = keep_container
+        self.cpu_num = cpu_num
+        self.common_server_created = False
+        self.run_args = {
+            'pylint': not no_pylint,
+            'flake8': not no_flake8,
+            'mypy': not no_mypy,
+            'tests': not no_test
+        }
+        self.pkgs = project_dir.split(',')
```

Comment:
need to specify that this param will support a list as well, as the name does not indicate it.

Reply:
done